### PR TITLE
replaced shortened inv = TRUE arguments with invert = TRUE (bugfix)

### PR DESCRIPTION
**Pull request by tscheypidi**

### CODE REVIEW

File: CRAN-RELEASE [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This package was submitted to CRAN on 2019-12-17.
+Once it is accepted, delete this file and tag the release (commit ebf7bcc9db).
```
Comment (pfuehrlich-pik) [inline CRAN-RELEASE:2]: Why do we need this file now?

File: R/toolISOhistorical.R [hunk 2]
```diff
@@ -367,17 +367,18 @@ toolISOhistorical <- function(m, mapping = NULL, additional_mapping = NULL, over
            m[getItems(mTr, dim = 1), getYears(mTr), ] <- ifelse(is.na(m[getItems(mTr, dim = 1), getYears(mTr), ]),
                                                                      mTr, m[getItems(mTr, dim = 1), getYears(mTr)])
          if (length(addM) > 0) {
-          m[is.na(m)] <- 0 }
+          m[is.na(m)] <- 0
+ }
```
Comment (pfuehrlich-pik) [inline R/toolISOhistorical.R:371]: broken indentation

File: CRAN-RELEASE (hunk 1 above)
Comment (tscheypidi) [reply CRAN-RELEASE]: we don't. I added it by accident. I removed it now again

File: R/toolISOhistorical.R (hunk 2 above)
Comment (tscheypidi) [reply R/toolISOhistorical.R]: fixed

**Review by pfuehrlich-pik — APPROVED** [review]

**Pull request merged**